docs/changelog/5_x.rst

```diff
@@ -0,0 +1,343 @@
+====
+v5.0
+====
+
+:edb-alt-title: EdgeDB v5
+
+To play with the new features, install the CLI using `our installation guide
+<https://www.edgedb.com/install>`_ and initialize a new project.
+
+.. code-block:: bash
+
+  $ edgedb project init
```

Comment:
```suggestion
  $ edgedb project init --server-version 5.0-beta.1
```
or whatever the name of this beta ends up being. I assume this is it but maybe I'm wrong.